v1/api/swagger/paths/images.yaml

```diff
@@ -0,0 +1,64 @@
+get:
+  tags:
+    - images
+  summary: List of all OS Images within an tenant 
+  description: >-
+    Returns an array of all OS images objects that have been created.
+    If GreenLake IAM issued token is used for authentication, 
+    then it is required to pass 'spaceid' header
+  operationId: Images_list
+  parameters:
+    - $ref: ../components/parameters/SpaceID.yaml
+    - $ref: ../components/parameters/Space.yaml
+  responses:
+    '200':
+      description: success
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              $ref: ../components/schemas/OSServiceImage.yaml
+    '401':
+      $ref: ../components/responses/Unauthorized.yaml
+    '500':
+      $ref: ../components/responses/InternalError.yaml
+
+post:
+  tags:
+    - images
+  summary: Create a new OS image
+  description: >-
+    Adds a new OS Image that can be referenced during host creation.
+    If GreenLake IAM issued token is used for authentication, then it is required 
+    to pass either 'spaceid' header. Note that Hoster or BMaaS Access Owner role is 
```

Comment:
What's the alternative option? Sentence says "to pass either...", but one 1 options specified (spaceid).

Reply:
Forgot to include the Space header (which is the space name).